javascript/javascript1/week1/age-ify.js

```diff
@@ -0,0 +1,6 @@
+const yearOfBirth = 1992;
+let yearFuture = 2045;
+console.log("typeof the variable 'yearOfBirth' is: " + typeof(yearOfBirth));
+console.log("typeof the variable 'yearFuture' is: " + typeof(yearFuture));
```

Comment:
you don't need to console the typeof variable because you already declared the year as number. so, no use of this statements.